src/OrchardCore/OrchardCore.ResourceManagement.Abstractions/ResourcesTagHelperProcessorContext.cs

```diff
@@ -0,0 +1,8 @@
+using System.IO;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace OrchardCore.ResourceManagement;
+
+public record ResourcesTagHelperProcessorContext(
+    TextWriter Writer,
+    ResourceTagType Type);
```

Comment:
```suggestion
public record ResourcesTagHelperProcessorContext(
    ResourceTagType Type,
    TextWriter Writer);
```

Reply:
Sure, but why?

Reply:
It makes more sense to specify type first